src/components/EditCoursePage/CollapsibleCourseRun.jsx

```diff
@@ -355,6 +351,24 @@ class CollapsibleCourseRun extends React.Component {
           </div>
         }
         <hr />
+        {type &&
+          <Field
+            name={`${courseId}.run_type`}
+            component={RenderSelectField}
+            options={currentFormValues.type ? courseRunTypeOptions[currentFormValues.type] : [{ label: 'Select Course Type first', value: '' }]}
+            extraInput={{ onInvalid: this.openCollapsible }}
+            label={
+              <FieldLabel
+                id="run_type.label"
+                text="Course Run Type"
+                helpText={(<div><p>TODO: Come up with run type helpText</p></div>)}
+              />
+            }
+            // Should this be disabled in other scenarios?
```

Comment:
Is this a comment we should address now? You can't edit after submission, right? Do we need that extraHelp here with a disable in that case?

Reply:
Talked about in person. We believe it should be be disabled as soon as any of the seats have a sku. Just need to write the filter function for that.